syntax = "proto3";



message SkillExecutedEvent 
{
  uint64 skill_id = 1;
  uint32 skill_table_id = 2;
  uint64 caster = 3;
  uint64 target = 4;
}